src/bundle/Resources/public/scss/mixins/_spinner.scss

```diff
@@ -0,0 +1,22 @@
+@mixin spinner($size, $line-size, $color: $ibexa-color-info) {
```

Comment:
That's backported from 4.6 so I left it as it is with hope there will be no conflicts, but I started to wonder if we should treat ibexa-color-info as default for spinner color - as I check it now on 4.6, we have 3 usages of info, 4 (or even 5 with this issue) of primary and one for white.

Do we want to leave it as it is for BC and counter-intuitive, or we'd rather to change it to primary? (which would require more work with upmerge, as these three places where we use info (as default) would need to be adjusted.